include/aspect/material_model/rheology/visco_plastic.h

```diff
@@ -316,6 +317,13 @@ namespace aspect
            */
           Rheology::ConstantViscosityPrefactors<dim> constant_viscosity_prefactors;
 
+          /**
+           * Object for computing the viscosity multiplied by a varying prefactor.
+           * This multiplication step is done just prior to calculating the effective
```

Comment:
I realize this is legacy documentation, but I think it actually should be removed as this is specific to how the visco_plastic model works, but this order of operations may be different when the function is called in other material models.